lib/i18n/tasks/translators/deepl_translator.rb

```diff
@@ -60,21 +60,102 @@ def options_for_plain
 
     # @param [String] value
     # @return [String] 'hello, %{name}' => 'hello, <i18n>%{name}</i18n>'
-    def replace_interpolations(value)
+    def original_replace_interpolations(value)
```

Comment:
If we don't use these functions, delete them. This can only cause unclear rebase conflicts later, and if we ever need them, they are still in the git history

Reply:
We have opposing ideas on what makes a rebase difficult :)
I think renaming makes the rebase easier because their code and our code are not mixed